docs/popup.js

```diff
@@ -253,23 +260,27 @@ function setUpPopup() {
         // Give the Marker room to be visible.
         divPosition.y -= 40;
       }
+      checkPresent(this.containerDiv, 'not present when drawing');
       this.containerDiv.style.left = divPosition.x + 'px';
       this.containerDiv.style.top = divPosition.y + 'px';
     }
 
     /** Sets the visibility to 'hidden'. */
     hide() {
+      checkPresent(this.containerDiv, 'not present when hiding');
       // The visibility property must be a string enclosed in quotes.
       this.containerDiv.style.visibility = 'hidden';
     }
 
     /** Sets the visibility to 'visible'. */
     show() {
+      checkPresent(this.containerDiv, 'not present when showing');
       this.containerDiv.style.visibility = 'visible';
     };
 
     /** @return {boolean} true if the popup is currently visible. */
     isVisible() {
+      checkPresent(this.containerDiv, 'not present when is visible');
```

Comment:
nit: slight trouble parsing this phrase - maybe 'not present when checking visible' ?